src/components/HydraRepl.tsx

```diff
@@ -1,7 +1,19 @@
-import React, { useState, useCallback } from 'react';
-import CodeMirror from '@uiw/react-codemirror';
 import { javascript } from '@codemirror/lang-javascript';
+import CodeMirror from '@uiw/react-codemirror';
 import { Play } from 'lucide-react';
+import React, { useState, useCallback } from 'react';
+
+// Test mode FFT generator constants
```

Comment:
Can we make sure to move constants to their own file/directory that benefits the organization of the repo?